cmd/cantool/main.go

```diff
@@ -163,7 +160,7 @@ func genGo(inputFile, outputFile string) error {
 	if err != nil {
 		return err
 	}
-	if err := ioutil.WriteFile(outputFile, output, 0600); err != nil {
+	if err := ioutil.WriteFile(outputFile, output, 0o600); err != nil {
```

Comment:
Can be replaced with `os.Writefile` in 1.16

Reply:
Though to allow other running an older version probably best to keep it as it is

Reply:
Let's give ourselves some time to transition over to 1.16 across the board, and then get at it! :smiley: